lib_nbgl/include/fonts/nbgl_font_rom.inc

```diff
@@ -18,24 +18,24 @@
 
 #ifdef SCREEN_SIZE_WALLET
 #ifdef TARGET_STAX
-#include "nbgl_font_inter_regular_24.inc"
-#include "nbgl_font_inter_semibold_24.inc"
-#include "nbgl_font_inter_medium_32.inc"
-#include "nbgl_font_inter_regular_24_1bpp.inc"
-#include "nbgl_font_inter_semibold_24_1bpp.inc"
-#include "nbgl_font_inter_medium_32_1bpp.inc"
+#include "fonts/nbgl_font_inter_regular_24.inc"
```

Comment:
Better change the makefile to add lib_nbgl/include/fonts to include path than adding path in all .c files

Reply:
Was easier than I though :) Done!